.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
 	"diffEditor.ignoreTrimWhitespace": true,
-	"files.autoSave": "afterDelay",
+	"files.autoSave": "off",
```

Comment:
@joselrio Why changing this?

Reply:
https://github.com/OutSystems/outsystems-ui/blob/0e39d1e65d22e5dd6bece6a86bd84c714584965e/.vscode/settings.json, 
With this option enabled, and when at development mode, files was being compiled after any change occurs at the file, almost at "onKeyboardChange"... too much compilations. To conclude... compiling will only occurs when you save the file, otherwise autoSave will trigger the compilation.